master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/ha/HARaftServer.java

```diff
@@ -69,71 +55,67 @@
 
 import org.apache.celeborn.common.RssConf;
 import org.apache.celeborn.common.haclient.RssHARetryClient;
+import org.apache.celeborn.common.util.ThreadUtils;
 import org.apache.celeborn.service.deploy.master.clustermeta.ResourceProtos;
 import org.apache.celeborn.service.deploy.master.clustermeta.ResourceProtos.ResourceResponse;
 
 public class HARaftServer {
-  private static final Logger LOG = LoggerFactory.getLogger(HARaftServer.class);
+  public static final Logger LOG = LoggerFactory.getLogger(HARaftServer.class);
 
-  private final int port;
-  private final InetSocketAddress masterRatisAddress;
-  private final org.apache.ratis.server.RaftServer server;
-  private final RaftGroupId raftGroupId;
-  private final RaftGroup raftGroup;
-  private final RaftPeerId raftPeerId;
+  public static final UUID CELEBORN_UUID =
+      UUID.nameUUIDFromBytes("CELEBORN".getBytes(StandardCharsets.UTF_8));
+
+  public static final RaftGroupId RAFT_GROUP_ID = RaftGroupId.valueOf(CELEBORN_UUID);
```

Comment:
it's hardcoded now, please let me know if anyone has concerns(have talked to some guys, and they do not object this removing)

Reply:
I think it's safe to remove.